pkg/controllers/managementuser/rbac/reconcile_roletemplate.go

```diff
@@ -170,3 +172,13 @@ func (m *manager) ensureGlobalResourcesRolesForPRTB(projectName string, rts map[
 
 	return sets.List(roles), nil
 }
+
+func hasReadVerbs(ruleVerbs []string) bool {
+	readVerbs := []string{"get", "list", "watch"}
+	for _, verb := range readVerbs {
+		if slice.ContainsString(ruleVerbs, verb) {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
Since we are already using the `"k8s.io/apimachinery/pkg/util/sets"` package we can simplify this creating a set with the verbs:

```go
var readVerbs = sets.New(
	"get", "list", "watch",
)
```

and use the HasAny method:

```go
hasAnyReadVerbs := readVerbs.HasAny(rule.Verbs...)
```